test/e2e/specs/site-editor/site-editor-inserter.spec.js

```diff
@@ -64,4 +65,199 @@ test.describe( 'Site Editor Inserter', () => {
 
 		await expect( blockLibrary ).toBeHidden();
 	} );
+
+	test( 'should open the inserter to patterns tab if using zoom out', async ( {
+		InserterUtils,
+	} ) => {
+		const zoomOutButton = InserterUtils.getZoomOutButton();
+		const inserterButton = InserterUtils.getInserterButton();
+		const blockLibrary = InserterUtils.getBlockLibrary();
+
+		await zoomOutButton.click();
+		await expect( await InserterUtils.getZoomCanvas() ).toBeVisible();
+
+		await inserterButton.click();
+		const patternsTab = InserterUtils.getBlockLibraryTab( 'Patterns' );
+		await expect( patternsTab ).toHaveAttribute(
+			'data-active-item',
+			'true'
+		);
```

Comment:
Should we use `aria-selected`? It's a more user perceivable attribute whereas `data-*` is purely for developers.

I checked to see if there was an `isSelected()` but there isn't.

You _can_ do this I believe but you'd have to modify your helper utils to accept options.

```
getByRole('button', {
selected: true,
});
```